tests/sentry/utils/kvstore/test_bigtable.py

```diff
@@ -0,0 +1,156 @@
+import itertools
+import os
+from typing import Iterator, Optional
+
+import pytest
+from google.oauth2.credentials import Credentials
+from sentry.utils.kvstore.abstract import K, KVStorage, V
+from sentry.utils.kvstore.bigtable import BigtableKVStorage
+
+
+@pytest.fixture
+def credentials() -> Credentials:
+    if "BIGTABLE_EMULATOR_HOST" not in os.environ:
+        pytest.skip(
+            "Bigtable is not available, set BIGTABLE_EMULATOR_HOST enironment variable to enable"
+        )
+
+    # The bigtable emulator requires _something_ to be passed as credentials,
+    # even if they're totally bogus ones.
+    return Credentials.from_authorized_user_info(
+        {key: "invalid" for key in ["client_id", "refresh_token", "client_secret"]}
+    )
```

Comment:
This duplicates the behavior in the `BigtableNodeStorage` tests but I expect that to be removed soon.